test/test_datasets_wildfire_frame_extractor.py

```diff
@@ -0,0 +1,166 @@
+import glob
+import tempfile
+import unittest
+
+from unittest.mock import patch
+
+from pathlib import Path
+
+import pandas as pd
+
+from pyronear.datasets.wildfire import FrameExtractor
+
+
+# TODO: test when only two frames available and when n_frames > count of available frames
+
+
+class WildFireFrameExtractorTester(unittest.TestCase):
+
+    @staticmethod
+    def download_video_fixtures(path_to_videos):
+        import urllib
+        import yaml
+        import pafy
```

Comment:
As the test will always be executed, better to move imports to the header!

Reply:
I put these libs in this well-defined namespace to make it clear which part of the code is going to connect and download videos elsewhere on the net. They may break in the future because of host policy, so we'll have to change.

It's not big deal if you prefer to strictly [follow the PEP 8](https://www.python.org/dev/peps/pep-0008/#imports), I was you trying to help the future us ;)

Reply:
I would have done the same thing if they were test-only requirements! But since they are also in the installation requirements and that Ctrl+F is quite user-friendly, I saw it as a cleaner option. But you're right, it's far from being critical haha

Reply:
Actually, `pafy` and `yaml` are test-only requirements and `urllib` is builtin. Ctrl-F doesn't give you information about namespace dependency. You may have missed my point. I don't want to spend more time on this, so go for it